packages/fiori/test/pages/NotificationListGroupItem.html

```diff
@@ -337,6 +337,11 @@ <h3>Events on ui5-list level</h3>
 			event.preventDefault();
 			notificationsPopover.showAt(event.detail.targetRef);
 		});
+
+		notificationListTop.addEventListener("itemClose", function (event) {
+				wcToastBS.textContent = event.detail.item.titleText;
+				wcToastBS.show();
+		});
```

Comment:
The event listeners in these samples are attached to the older events which were renamed from camel cased names to kebab case names. Can you update them all, instead of attaching to an outdated event name?